src/StrawberryfieldUtilityService.php

```diff
@@ -332,7 +284,50 @@ public function verifyDrush(string $execpath, string $home) :bool {
     return $canexecute;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getCountByProcessorInSolr(EntityInterface $entity, string $processor, array $indexes = [], string $checksum = ''): int {
+    $count = 0;
+
+    // If no index specified, query all of them.
+    if (empty($indexes)) {
+      $indexes = \Drupal::entityTypeManager()
+        ->getStorage('search_api_index')
+        ->loadMultiple();
```

Comment:
Since this function is very specific to our Data Source and the fields we have there, it would make more sense to only use indexes where the data source (Strawberry Flavour Data Source) is setup to be indexed. There is some example code in SB Runners for that.

Reply:
Note: not a blocker. Since there is a filter of course for the data source in the query.

Reply:
Makes sense, I've taken the chance to sort out a little bit the datasource class